### PR TITLE
Added Insertion Sort CPP fixes #8

### LINKED ISSUE #8: Insertion Sort in C++

### PR DESCRIPTION
**Pull request by hnishal**

### CODE REVIEW

File: Sorting/insertion_sort/insertionSort.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        {  
+            arr[j + 1] = arr[j];  
+            j--;  
+        }  
+        arr[j + 1] = temp; 
+	}
+}
+void display(int arr[],int size){
+	for(int i=0;i<size;i++){
+		cout<<arr[i]<<"\t";
```
Comment (nikochiko) [inline Sorting/insertion_sort/insertionSort.cpp:17]: Can we use spaces here instead of tabs?

**Review by nikochiko — CHANGES_REQUESTED** [review]
Great work! Just one minor change and this is good to go. Can you also change the filename to `insertion_sort.cpp` (notice the snake case) for the sake of convention.

File: Sorting/insertion_sort/insertionSort.cpp (hunk 1 above)
Comment (hnishal) [reply Sorting/insertion_sort/insertionSort.cpp:17]: yeah, my bad

**Comment by nikochiko** [comment]
Hey @hnishal the code is good! Why did you close it?
You can just add changes on top of the last one

**Comment by hnishal** [comment]
I'm really new to this, didnt know how to proceed with that

**Comment by nikochiko** [comment]
Oh, no problem. You can refer this for git and github - https://www.dataschool.io/how-to-contribute-on-github/

**Pull request closed without merging**